app/create/index.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+var Angular = require('angular');
+var State = require('./state');
```

Comment:
nit:  new line between these. to separate package imports from local file imports